internal/protocols/ethereum/signer_test.go

```diff
@@ -137,3 +158,57 @@ func TestSigner_Sign(t *testing.T) {
 		})
 	}
 }
+
+func Test_validatePrivateKeyType(t *testing.T) {
+	assert := assert.New(t)
+	type args struct {
+		pk crypto.PrivateKey
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *ecdsa.PrivateKey
+		wantErr bool
+	}{
+		{
+			"success-secp256k1-charlotte",
+			args{
+				secp256k1test.CharlottePrivateKey,
+			},
```

Comment:
This unit tests will only cover one of the lines from 
```
	case secp256k1.PrivateKey:
		return pk.ECDSA()
	case *secp256k1.PrivateKey:
		return pk.ECDSA()
```
Can you see if you can make another one to cover the other line. Thanks :)